docs/source/docker/getting_started/selection.rst

```diff
@@ -0,0 +1,518 @@
+.. _worker-selection:
+
+Selection
+=========
+
+Lightly allows you to specify the subset to be selected based on several objectives.
+
+E.g. you can specify that the images in the subset should be visually diverse, be images the model struggles with (Active Learning),
+should only be sharp images, or have a certain distribution of classes, e.g. be 50% from sunny, 30% from cloudy and 20% from rainy weather.
+
+Each of these objectives is defined by a `strategy`. A strategy consists of two parts:
+
+- The :code:`input` defines which data the objective is defined on. This data is either a scalar number or a vector for each sample in the dataset.
+- The :code:`strategy` itself defines the objective to apply on the input data.
+
+Lightly allows you to specify several objectives at the same time. The algorithms try to fulfil all objectives simultaneously.
+
+Lightly's data selection algorithms are supporting four types of input:
+
+- **Embeddings** computed using `our open source framework for self-supervised learning <https://github.com/lightly-ai/lightly>`_
+- **Lightly metadata** are metadata of images like the sharpness and computed out of the images themselves by Lightly.
+- (Optional)  :ref:`Model predictions <docker-datasource-predictions>` such as classifications, object detections or segmentations
+- (Optional) :ref:`Custom metadata <docker-datasource-metadata>` can be anything you can encode in a json file (from numbers to categorical strings)
+
+Prerequisites
+-------------
+
+For using the selection, you must have
```

Comment:
the selection? I'd rephrase: "In order to use the selection feature, ..."

also must is a bit of a strong word

Reply:
I changed it to `In order to use the selection feature, you need to`